corehq/apps/reports/views.py

```diff
@@ -640,6 +628,44 @@ def calculate_day(interval, day, day_change):
     return day
 
 
+def soft_shift_to_domain_timezone(instance: ReportNotification):
+    time_difference = get_timezone_difference(instance.domain)
```

Comment:
The only difference with `soft_shift_to_server_timezone` seems to be `recalculate_hour` and `instance.minute`. Would you consider making these DRY?